dashboard/graphql/view.py

```diff
@@ -18,6 +18,7 @@
 
 
 class DashboardGraphQLView(LoginRequiredMixin, GraphQLView):
+    execution_context_class = PromiseExecutionContext
```

Comment:
Did not find execution_context_class used anywhere in the file.

Reply:
This is needed as part of the promise support using the library `graphql-core-promise` for graphql. I have more info on this in the PR description